### PR TITLE
Partial (1/2) spanish translation ch08-02-strings.md

### PR DESCRIPTION
**Pull request by jossue33**
Number of words: 1072
Line: 1-195

### CODE REVIEW

File: second-edition/src/ch08-02-strings.md [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
-These string types can store text in different encodings or be represented in
-memory in a different way, for example. We won’t discuss these other string
-types in this chapter; see their API documentation for more about how to use
-them and when each is appropriate.
-
-### Creating a New String
-
-Many of the same operations available with `Vec<T>` are available with `String`
-as well, starting with the `new` function to create a string, shown in Listing
+## Tienda de cadenas de caracteres de texto codificado UTF-8
```
Comment (thecodix) [inline second-edition/src/ch08-02-strings.md]: 'Los strings almacenan texto codificado en UTF-8'

Please consider reviewing

**Comment by thecodix** [comment]
hey @jossue33  would you mind sticking to the line length of 120 characters? (around)

It makes way easier tasks of reviewing :)

Thanks!

**Comment by jossue33** [comment]
@thecodix todo listo.

**Comment by thecodix** [comment]
Sorry to bother you again, you mind adjusting the line length to have aprox the same line length as the english translation?

It makes easier for the review / edition tasks later

Thanks a lot @jossue33 !!

**Comment by jossue33** [comment]
ya esta espero así sea.

**Comment by thecodix** [comment]
machine translated

**Pull request merged**